esi_leap/common/idp/dummyIDP.py

```diff
@@ -0,0 +1,79 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import sys
+
+from keystoneclient.v3.projects import Project
+from keystoneclient.v3.projects import ProjectManager
+from keystoneclient.v3.users import User
+from keystoneclient.v3.users import UserManager
+from oslo_utils import uuidutils
+
+from esi_leap.common import exception
+from esi_leap.common.idp import baseIDP
+
+
+class DummyIDP(baseIDP.BaseIDP):
+
+    dummy_project_list: list[Project]
```

Comment:
I would make both of these dictionaries where the key is the `id`. This simplifies `get_project_name` and `_get_project_obj_from_id` significantly.